_includes/helper_action_functions.md

```diff
@@ -0,0 +1,25 @@
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|add_tags      |strings[]     |An array of tag names corresponding to previously created tags to add to this person when it is created.
+|add_tags_uri  |strings[]     |An array of tag URIs corresponding to previously created tags to add to this person when it is created.
+|add_lists     |strings[]     |An array of list names corresponding to previously created lists to add to this person when it is created.
+|add_lists_uri     |strings[]     |An array of list URIs corresponding to previously created lists to add to this person when it is created.
+|add_questions_responses_uri      |[AnswerActionFunction](#answer-action-function-fields)     |An array of Person Signup Helper Answer objects, indicating answers which should be applied to the Person at the moment of signup
```

Comment:
Heads up on the "person signup" bit here, I think this will be included in places besides the person signup helper.